the-chemical-society-of-japan.csl

```diff
@@ -0,0 +1,189 @@
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="article-journal" match="any">
+        <text variable="container-title" form="short" font-style="italic" suffix=" "/>
+      </if>
+      <else-if type="book" match="any">
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else-if type="chapter" match="any">
+        <text term="in" suffix=" "/>
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else>
+        <text variable="container-title" font-style="normal"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <number suffix=" " variable="edition" form="ordinal"/>
+    <label plural="never" variable="edition" form="short"/>
+  </macro>
+  <macro name="event">
+    <text variable="event-place" suffix=", "/>
+    <date variable="event-date">
+      <date-part name="month" range-delimiter="" suffix=" "/>
+      <date-part name="day" range-delimiter="" suffix=", "/>
+      <date-part name="year" range-delimiter="" font-weight="bold"/>
+    </date>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book article" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <date date-parts="year" form="numeric" variable="issued" font-weight="normal">
+      <date-part name="year" font-weight="bold"/>
+    </date>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="verb-short" plural="never" suffix=" "/>
+      <name initialize="false" initialize-with=". "/>
+    </names>
+  </macro>
+  <macro name="locators">
+    <choose>
+      <if type="article-journal" match="any">
+        <number font-style="italic" variable="volume"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <text term="volume" form="short" text-case="capitalize-first" font-style="normal" suffix=" "/>
+        <number suffix=", " variable="volume"/>
+        <text term="chapter" form="short" text-case="capitalize-first" suffix=" "/>
+        <number variable="chapter-number"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="page">
+    <choose>
+      <if type="article-journal" match="any">
+        <text variable="page-first"/>
+      </if>
+      <else-if type="chapter" match="any">
+        <label suffix=" " variable="page" form="short"/>
+        <text variable="page"/>
+      </else-if>
+      <else>
+        <text variable="page"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher" suffix=", "/>
+    <text variable="publisher-place"/>
+  </macro>
+  <citation collapse="citation-number">
```

Comment:
You'll want to add

```
<sort>
  <key variable="citation-number"/>
</sort>
```
here, too. Otherwise you could be getting something like `5,3` for subsequent citations.